tests/fastnumberpathtests.cpp

```diff
@@ -0,0 +1,134 @@
+#include <cassert>
+#include <cinttypes>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "simdjson/jsonparser.h"
+
+static inline uint64_t splitmix64_stateless(uint64_t index) {
```

Comment:
Speaking as a reasonably intelligent programmer not at all steeped in the nuances of bit twiddling, it took me some googling to understand what this was for, since the method doesn't really say anything about it being a random number seed and the method certainly doesn't self-explain :)

It'd be worthwhile to comment this method the same way [this blog post did](https://bduvenhage.me/rng/2019/04/06/the-intel-drng.html):

//! Stateless [0,2^64) splitmix64 by Daniel Lemire https://github.com/lemire/testingRNG . Useful for seeding RNGs.

Reply:
You are right, of course.